src/card.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import london from "./london.jpg";
+import Glasgow from "./Glasgow.jpg";
+import manchester1 from "./manchester1.jpeg";
+function TouristInfoCards() {
+  return (
+    <div id="allCard">
+      <div className="card">
+        <h3>London</h3>
+        <img src={london} className="card-img-top" id="london" />
+        <div className="card-body">
+          <p>
+            London is the best city in the world , you can do anything there
+            such as
+          </p>
+          <a
+            href="https://www.cityoflondon.gov.uk/things-to-do"
+            className="btn btn-primary"
+          >
+            More Information
+          </a>
+        </div>
+      </div>
+      <div className="card">
+        <h3>Glasgow</h3>
+        <img src={Glasgow} className="card-img-top" id="glasgow" />
+        <div className="card-body">
+          <p>
+            Glasgow is the best city in the world , you can do anything there
+            such as
+          </p>
+          <a href="https://www.glasgow.gov.uk/" className="btn btn-primary">
+            More Information
+          </a>
+        </div>
+      </div>
+      <div className="card">
+        <h3>Manchester</h3>
+        <img src={manchester1} className="card-img-top" id="manchester" />
+        <div className="card-body">
+          <p>
+            Manchester is the best city in the world , you can do anything there
+            such as
+          </p>
+          <a href="https://www.manchester.gov.uk/" className="btn btn-primary">
+            More Information
+          </a>
```

Comment:
Can you try to add all the cities into an array and then use make to display them, instead of having this much HTML for every card? You only have one card inside the map instead!

Reply:
You mean like  bring data from API  ?

Reply:
No not an API, but just local array of objects and add your summary, link, and image and then loop them through using map to display them as a cards.
Something like this
Let cities =[
{
Name: "London",
More_info:"add a link",
Image_link: "https://....",
Summary:"London is wonderful city..."
}]